icecreamshop/icecreamshop/src/main/java/com/summerschool/icecreamshop/repository/RateRepository.java

```diff
@@ -1,6 +1,9 @@
 package com.summerschool.icecreamshop.repository;
 
-import org.springframework.data.repository.CrudRepository;
+import com.summerschool.icecreamshop.model.Rate;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
 
-public interface RateRepository extends CrudRepository<Integer, RateRepository> {
+@Repository
+public interface RateRepository extends JpaRepository<Rate, Integer> {
```

Comment:
Please make sure that the ID parameter has the same type as the ID property inside your Rate Entity, meaning that you should have Long instead of Integer.